packages/compass-crud/src/components/update-data-menu.tsx

```diff
@@ -31,39 +31,15 @@ const UpdateMenuButton: React.FunctionComponent<UpdateMenuButtonProps> = ({
   );
 };
 
-type UpdateMenuProps = UpdateMenuButtonProps & {
-  disabledTooltip: string;
-};
-
-const UpdateMenu: React.FunctionComponent<UpdateMenuProps> = ({
+const UpdateMenu: React.FunctionComponent<UpdateMenuButtonProps> = ({
   isWritable,
   onClick,
-  disabledTooltip,
 }) => {
-  if (isWritable) {
-    return (
-      <UpdateMenuButton isWritable={true} onClick={onClick}></UpdateMenuButton>
-    );
-  }
-
   return (
-    <Tooltip
-      trigger={({
-        children: tooltipChildren,
-        ...tooltipTriggerProps
-      }: React.HTMLProps<HTMLInputElement>) => (
-        <div {...tooltipTriggerProps}>
-          <UpdateMenuButton onClick={onClick} isWritable={false} />
-          {tooltipChildren}
-        </div>
-      )}
-      // Disable the tooltip when the instance is in a writable state.
-      isDisabled={isWritable}
-      justify="middle"
-      delay={500}
-    >
-      {disabledTooltip}
-    </Tooltip>
+    <UpdateMenuButton
+      isWritable={isWritable}
+      onClick={onClick}
+    ></UpdateMenuButton>
```

Comment:
We don’t end up including the limit/skip values in the update/delete (we can’t, right), correct? Do we need to do anything to make users aware of that?

Reply:
Like I said in slack - the count will take into account the limit and skip and therefore be incorrect in the modal so I don't think we should be doing this.